File: Program's_Contributed_By_Contributors/Java_Programs/PALINDROME.java
import java.util.*;
public class Main
{
	public static void main(String[] args) {
	    Scanner sc=new Scanner(System.in);
		System.out.println("enter n");
		int nn=sc.nextInt();
		for(int i=1;i<=nn;i++){
		    System.out.println("Enter number");
		    int num=sc.nextInt();
		    String newnum="";
		    int n=num;
		    int d;
		    while(n!=0){
		         d=n%10;
		         n=n/10;
		         newnum = newnum+d;
		    }
		    if(num==0 || num/10==0){
		         System.out.println("not palindrome");
		    }
		    else if(Integer.valueOf(newnum) == num){
		        System.out.println("Palindrome");
		    }
		    else{
		        System.out.println("not palindrome");
		    }
		}
	}
}

File: Program's_Contributed_By_Contributors/Python_Programs/palindrome.py
# initialize the variables
num=int(input("Enter number:"))
org=num
rev=0

# creating a list to count the occurence 
l1=list()
l2=list()

# main logic to check integer palindrome
while(num>0):
    rem=num%10
    rev=rev*10+rem
    num=num//10

    l2.append(rem)
    if rem not in l1:
        l1.append(rem)
   

# check the integer is a palindrome ?
if(org==rev):
    print("The number is a palindrome!")
else:
    print("The number is not a palindrome!")
    
# print the number of occurence
for i in l1:
    print(f"Digit {i}: {l2.count(i)} occurrences")
    